src/rtde/rtde_client.cpp

```diff
@@ -83,8 +88,8 @@ bool RTDEClient::init(const size_t max_num_tries, const std::chrono::millisecond
     if (client_state_ == ClientState::INITIALIZED)
       return true;
 
-    URCL_LOG_ERROR("Failed to initialize RTDE client, retrying in 10 seconds");
-    std::this_thread::sleep_for(std::chrono::seconds(10));
+    URCL_LOG_ERROR("Failed to initialize RTDE client, retrying in 1 seconds");
+    std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
Initially the idea was to avoid a long waiting period when some variables of the output recipe were unavailable. But this does not make sense to break the use case you're describing. Reverted in b9f848477f0013e187d26e22593608b5b0faad95.